packages/cli/generators/controller/index.js

```diff
@@ -183,9 +183,9 @@ module.exports = class ControllerGenerator extends ArtifactGenerator {
       },
       {
         type: 'confirm',
-        name: 'idRequired',
-        message: 'Is the id required when creating an instance?',
-        default: false,
+        name: 'idOmitted',
+        message: 'Is the id omitted when creating an instance?',
```

Comment:
Nitpick: I would use "creating a new instance" to make it even more obvious that this prompt is related to new instances only